builtin/credential/approle/path_role.go

```diff
@@ -51,12 +51,18 @@ type roleStorageEntry struct {
 	// A constraint, if set, requires 'secret_id' credential to be presented during login
 	BindSecretID bool `json:"bind_secret_id" structs:"bind_secret_id" mapstructure:"bind_secret_id"`
 
-	// A constraint, if set, specifies the CIDR blocks from which logins should be allowed
+	// Deprecated: A constraint, if set, specifies the CIDR blocks from which logins should be allowed
 	BoundCIDRListOld string `json:"bound_cidr_list,omitempty"`
 
-	// A constraint, if set, specifies the CIDR blocks from which logins should be allowed
+	// Deprecated: A constraint, if set, specifies the CIDR blocks from which logins should be allowed
 	BoundCIDRList []string `json:"bound_cidr_list_list" structs:"bound_cidr_list" mapstructure:"bound_cidr_list"`
 
+	// A constraint, if set, specifies the CIDR blocks from which logins should be allowed
+	SecretIDBoundCIDRs []string `json:"secret_id_bound_cidrs" structs:"bound_cidr_list" mapstructure:"bound_cidr_list"`
```

Comment:
I did that because the `BoundCIDRList` above is sort of funny. Its json tag is `bound_cidr_list_list`, but the rest of its tags are `bound_cidr_list`. I thought perhaps to be safe, I should use the same mapping and struct tags as it. 

I grepped the package and couldn't find any use of `structs.New(role)` or `structs.Map(role)`, so I was tempted to strip all the `struct` and `mapstructure` tags altogether, but I didn't want to go too far. Though `roleStorageEntry` isn't exported, I wondered if someone might have a fork somewhere where I shouldn't mess with how it's sent to struct/map. 

TLDR; I was thrown by the inconsistent `BoundCIDRLIst` tags. :-)

Reply:
Yeah, I omit the structs and mapstructure tags by default at this point.  They probably aren't necessary and can likely be removed.

Reply:
Huh, I just merged in master and someone when through and stripped the struct tags from this object, so I did too. Also updated my mapstructure tag to match the json tag.